clients/include/testing_logging.hpp

```diff
@@ -71,15 +78,36 @@ void testing_logging()
 {
     rocblas_pointer_mode test_pointer_mode = rocblas_pointer_mode_host;
 
-    // set environment variable ROCBLAS_LAYER = 1 to turn on logging. Note that putenv
+    // set environment variable ROCBLAS_LAYER to turn on logging. Note that putenv
     // only has scope for this executable, so it is not necessary to save and restore
-    // this environment variable for the next executable
-    //  char env_string[80] = "ROCBLAS_LAYER=1";
-    //  char env_string[80] = "ROCBLAS_LAYER=2";
-    char env_string[80] = "ROCBLAS_LAYER=3";
-    verify_equal(putenv(env_string), 0, "failed to set environment variable ROCBLAS_LAYER=3");
+    // this environment variable
+    //
+    // ROCBLAS_LAYER is a bit mask:
+    // ROCBLAS_LAYER = 1 turns on log_trace
+    // ROCBLAS_LAYER = 2 turns on log_bench
+    // ROCBLAS_LAYER = 3 turns on log_trace and log_bench
+    char env_rocblas_layer[80] = "ROCBLAS_LAYER=3";
+    verify_equal(
+        putenv(env_rocblas_layer), 0, "failed to set environment variable ROCBLAS_LAYER=3");
+
+    // set environment variable to give pathname of for log_trace file
+    std::string trace_name1             = "stream_trace.csv";
+    char env_rocblas_log_trace_path[80] = "ROCBLAS_LOG_TRACE_PATH=stream_trace.csv";
+    verify_equal(putenv(env_rocblas_log_trace_path),
+                 0,
+                 "failed to set environment variable ROCBLAS_LAYER=3");
+
+    // set environment variable to give pathname of for log_bench file
+    std::string bench_name1             = "stream_bench.txt";
+    char env_rocblas_log_bench_path[80] = "ROCBLAS_LOG_BENCH_PATH=stream_bench.txt";
+    verify_equal(putenv(env_rocblas_log_bench_path),
+                 0,
+                 "failed to set environment variable ROCBLAS_LAYER=3");
```

Comment:
ROCBLAS_LAYER=3 should be ROCBLAS_LOG_BENCH_PATH=stream_bench.txt

Reply:
Thanks, good catch, will fix